### PR TITLE
Add support for local registry

### LINKED ISSUE #12: Add support for local registry
- Enabling support for the usage of local registry

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
Fixes: #12
Signed-off-by: Prajyot Parab <prajyot.parab@ibm.com>

### CODE REVIEW

File: playbooks/roles/ocp-config/README.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ Requirements
 
  - All the required configurations are done on the bastion node eg: HTTP, openshift-install binary is available on $PATH. This can also be achieved by using [ocp4-helpernode](https://github.com/RedHatOfficial/ocp4-helpernode) playbook.
  - Master count can be extracted from host group 'masters'.
+ - Updated pull secret file with registry credentials located at `~/.openshift/pull-secret-updated` (Required for restricted network install)
+ - Registry certificate file located at `/opt/registry/certs/domain.crt` (Required for restricted network install)
+ - Local registry with local repo named as `ocp4/openshif4` (Required for restricted network install)
```
Comment (bpradipt) [inline playbooks/roles/ocp-config/README.md]: typo? **openshif4**

File: playbooks/roles/ocp-config/templates/install-config.yaml.j2 [hunk 2]
```diff
@@ -22,8 +22,21 @@ networking:
...
+pullSecret: '{{ lookup('file', '~/.openshift/pull-secret-updated') }}'
+additionalTrustBundle: |
+{{ lookup('file', '/opt/registry/certs/domain.crt') | indent(2, first=True) }}
+imageContentSources:
+- mirrors:
+  - registry.{{ install_config.cluster_id }}.{{ install_config.cluster_domain }}:5000/ocp4/openshift4
+  source: quay.io/openshift-release-dev/ocp-release-nightly
+- mirrors:
+  - registry.{{ install_config.cluster_id }}.{{ install_config.cluster_domain }}:5000/ocp4/openshift4
+  source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
```
Comment (bpradipt) [inline playbooks/roles/ocp-config/templates/install-config.yaml.j2:36]: Is this the right source ?

Comment (Prajyot-Parab) [reply playbooks/roles/ocp-config/templates/install-config.yaml.j2:36]: Fixed. Thanks, @bpradipt for pointing this out.

**Review by bpradipt — APPROVED** [review]
lgtm

**Review by yussufsh — APPROVED** [review]

**Pull request merged**